recipes/opentdf-client/all/conanfile.py

```diff
@@ -70,6 +70,7 @@ def requirements(self):
         self.requires("ms-gsl/2.1.0")
         self.requires("nlohmann_json/3.11.1")
         self.requires("jwt-cpp/0.4.0")
+        self.requires("magic_enum/0.8.2")
```

Comment:
Has this requirement always been necessary? Or is it just from 1.4.0 onwards? If so, a version check to conditionally add it would be better :)